build/metaschema/json/pull-datatypes.xsl

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+    xmlns:xs="http://www.w3.org/2001/XMLSchema"
+    exclude-result-prefixes="xs"
+    xpath-default-namespace="http://www.w3.org/2001/XMLSchema"
+    version="3.0" xmlns="http://www.w3.org/2005/xpath-functions"
+     expand-text="true">
+    
+    <xsl:output indent="yes"/>
+        
+    <xsl:strip-space elements="*"/>
+    
+    <xsl:preserve-space elements="documentation"/>
+    <!--Converts file:/C:/Users/wap1/Documents/OSCAL/build/metaschema/xml/oscal-datatypes.xsd in JSON Schema notation-->
```

Comment:
Not sure why this path is in here. This needs to be cleaned up.

Reply:
Stray comment now cleaned up.